service/Service/appsettings.Development.json.old

```diff
@@ -1,131 +0,0 @@
-{
-  "KernelMemory": {
-    "Service": {
-      "RunWebService": true,
-      "RunHandlers": true,
-      "OpenApiEnabled": true
-    },
-    "ContentStorageType": "SimpleFileStorage",
-    "TextGeneratorType": "AzureOpenAIText",
-    "ServiceAuthorization": {
-      "Enabled": false,
-      "AuthenticationType": "APIKey",
-      "HttpHeaderName": "Authorization",
-      "AccessKey1": "",
-      "AccessKey2": ""
-    },
-    "DataIngestion": {
-      "OrchestrationType": "Distributed",
-      "DistributedOrchestration": {
-        "QueueType": "SimpleQueues"
-      },
-      "EmbeddingGenerationEnabled": true,
-      "EmbeddingGeneratorTypes": [
-        "AzureOpenAIEmbedding"
-      ],
-      "MemoryDbTypes": [
-        "Postgres"
-      ],
-      "ImageOcrType": "AzureAIDocIntel",
-      "DefaultSteps": [
-        "extract",
-        "partition",
-        "gen_embeddings",
-        "save_records"
-      ]
-    },
-    "Retrieval": {
-      "MemoryDbType": "Postgres",
-      "EmbeddingGeneratorType": "AzureOpenAIEmbedding",
-      "SearchClient": {
-        "MaxAskPromptSize": -1,
-        "MaxMatchesCount": 100,
-        "AnswerTokens": 300,
-        "EmptyAnswer": "INFO NOT FOUND"
-      }
-    },
-    "Services": {
-      "Postgres": {
-        "ConnectionString": "Host=localhost;Port=5432;Username=delucato;Password=",
-        "TableNamePrefix": "km-"
-      },
-      "SimpleQueues": {
-        "Directory": "_queues"
-      },
-      "SimpleVectorDb": {
-        "Directory": "_vectors",
-        "StorageType": "Volatile"
-      },
-      "SimpleTextDb": {
-        "Directory": "_textdb",
-        "StorageType": "Volatile"
-      },
-      "SimpleFileStorage": {
-        "Directory": "_files"
-      },
-      "AzureBlobs": {
-        "Auth": "ConnectionString",
-        "Container": "smemory",
-        "Account": "devisstorage",
-        "ConnectionString": "DefaultEndpointsProtocol=https;AccountName=devisstorage;AccountKey=DY85QZ0UzKR4IRHaNMly4vUwz2e24hH94OyM2kvPvVBg239Jr7v77MLpni6gr0qOWozC/dFESXAO+AStuZHmSw==;EndpointSuffix=core.windows.net"
-      },
-      "AzureAISearch": {
-        "Auth": "ApiKey",
-        "Endpoint": "https://devisazuresearch.search.windows.net",
-        "APIKey": "SYDpZBS16nmvNXEILH0bMFPUPOuc3SSeDRwHbSjUqMAzSeB7McDR"
-      },
-      "OpenAI": {
-        "TextModel": "gpt-3.5-turbo-16k",
-        "EmbeddingModel": "text-embedding-ada-002",
-        "APIKey": "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",
-        "OrgId": "",
-        "MaxRetries": 10
-      },
-      "AzureQueues": {
-        "Auth": "ConnectionString",
-        "Account": "devisstorage",
-        "ConnectionString": "DefaultEndpointsProtocol=https;AccountName=devisstorage;AccountKey=DY85QZ0UzKR4IRHaNMly4vUwz2e24hH94OyM2kvPvVBg239Jr7v77MLpni6gr0qOWozC/dFESXAO+AStuZHmSw==;EndpointSuffix=core.windows.net"
```

Comment:
@dluc - these aren't my credentials in here so whoever pushed this should probably cycle their keys lol.